src/internal/faucet.ts

```diff
@@ -35,7 +35,11 @@ export async function fundAccount(args: {
 
   const txnHash = data.txn_hashes[0];
 
-  await waitForTransaction({ aptosConfig, transactionHash: txnHash, options: { timeoutSecs } });
+  await waitForTransaction({
```

Comment:
probably can be removed after https://github.com/aptos-labs/aptos-ts-sdk/pull/110#pullrequestreview-1693839440